great-idea/css/index.css

```diff
@@ -63,4 +63,258 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container {
+
+    max-width: 880px;
+	margin: 0 auto;
+}
+.flex-container{
+	display:flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	align-items: center;
+    word-spacing: 50px;
+    margin-top: 65px;
+	padding-bottom: 100px;
+}
+.flex-container nav a{
+	font-size: 14px;  
+	color: grey;
+	margin-right: 0px;
+	text-decoration: none;
+
+}
+.flex-container nav a:first-child {
+	margin-left: 0;
+  }
+  
+.flex-container nav a:last-child {
+	margin-right: 50px;
+}
+.flex-container nav a:hover{
+	text-decoration: underline;
+}
+.flex-container1{
+	display: flex;
+	flex-wrap: row top;
+	font-size: 77px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 75px;
+	margin-bottom: 100px;
+	vertical-align: center;
+	
+}
+.flex-container1 .flex-container2 button{
+	font-size: 12px;
+	padding: 6px 28px;
+	background: transparent;
+	border: 1px black solid;
+	cursor: pointer;
+}
+.flex-container1 .flex-container2 button:hover {
```

Comment:
Try to be a little more descriptive on the class names.  Having more distinct names can help with the reading.